### DISCUSSION TITLE
😂 引擎示例 😂

### DISCUSSION THREAD

**Discussion by imohuan** [layabox/LayaAir#1360] [General]
# http://layaair2.ldc2.layabox.com/demo/?category=2d&group=Skeleton&name=ChangeSkin
# 你们这个实例demo太简陋了，怎么用都不知道，能不能对我们初学者一点动力啊，看个教程一来就要200多
# 怎么用，能不能帮一下啊，我看了一下午都没有看懂你这示例怎么弄的，看了github也没看懂，代码放哪里啊，社区里面也问了，人都没有一个，说实话这社区没人去回答
# 所以能不能问问作者，这么弄啊

**Comment by imohuan** [comment]
鬼都没有一个

**Comment by lirongf** [comment]
我们尽快出一批如何进行示例学习的教程，更方便你们来使用示例下载demo

**Comment by lirongf** [comment]
你下好git 用vscode打开   能看到所有的示例

**Comment by cupen** [comment]
@imohuan  大哥您消消气, 字体改用小一号的正常字体.  如果 laya 谁欠你钱可以告诉我, 我来帮你讨.  相信我,讨债很厉害的.

**Comment by LayaCharley** [comment]
LayaAir的免费视频教程地址在这里：https://space.bilibili.com/1736809941
也可以使用IDE创建更多示例DEMO，LayaAir3增加了很多丰富的示例DEMO，或者您去资源商店，也有免费的项目源码供您学习的下载https://store.layaair.com/
